pacai/ui/captureGraphicsDisplay.py

```diff
@@ -713,83 +111,6 @@ def debugDraw(self, cells, color=[1.0, 0.0, 0.0], clear=False):
             if self.frameTime < 0:
                 graphicsUtils.refresh()
 
-    def clearExpandedCells(self):
-        if 'expandedCells' in dir(self) and len(self.expandedCells) > 0:
-            for cell in self.expandedCells:
-                graphicsUtils.remove_from_screen(cell)
-
```

Comment:
Did you remember to pull for merge conflicts? I believe this updateDistributions has already been removed. Just make sure to pull from develop before passing this to on.

Reply:
Pulled from linqs/develop and fixed merge conflicts.

Reply:
You are right. I removed updateDistributions and drawDistributions.